### PR TITLE
[ALLUXIO-2847] Metrics followup

### PR DESCRIPTION
**Pull request by aaudiber**
https://alluxio.atlassian.net/browse/ALLUXIO-2847

Followup to https://github.com/Alluxio/alluxio/pull/5530

This PR updates the metrics documentation and adds corresponding support for ufs writes

### CODE REVIEW

File: core/server/worker/src/main/java/alluxio/worker/netty/DataServerUfsFileWriteHandler.java [hunk 1]
```diff
@@ -47,16 +48,21 @@
...
       mUfsPath = createUfsFileOptions.getUfsPath();
-      mUnderFileSystem = mUfsManager.get(createUfsFileOptions.getMountId()).getUfs();
+      Ufs ufs = mUfsManager.get(createUfsFileOptions.getMountId());
+      mUnderFileSystem = ufs.getUfs();
       mOutputStream = mUnderFileSystem.create(mUfsPath,
           CreateOptions.defaults().setOwner(createUfsFileOptions.getOwner())
               .setGroup(createUfsFileOptions.getGroup())
               .setMode(new Mode((short) createUfsFileOptions.getMode())));
+      String ufsName = ufs.getUfsUri().toString().replace("/", "_");
+      String metricName = String.format("BytesWrittenUfs-Ufs:%s", ufsName);
```
Comment (calvinjia) [inline core/server/worker/src/main/java/alluxio/worker/netty/DataServerUfsFileWriteHandler.java]: The logic to convert from a `Ufs` to the metric name should reside either in the `Ufs` or some utils method?

File: docs/en/Metrics-System.md [hunk 2]
```diff
@@ -108,10 +118,8 @@ The following shows the details of the available metrics.
 * BlocksDeleted: Total number of blocks deleted.
 * BlocksEvicted: Total number of blocks evicted.
 * BlocksPromoted: Total number of blocks promoted.
-* NettyBlockRead: Total number of netty block read request to the worker.
-* NettyBlockReadFailures: Total number of netty block read request failed on the worker.
-* NettyBlockWrite: Total number of netty block write request to the worker.
-* NettyBlockWriteFailures: Total number of netty block write request failed on the worker.
+* BytesReadUfs-UFS:${UFS}: Total number of bytes read from the specified UFS on this worker
+* BytesWrittenUfs-UFS:${UFS}: Total number of bytes written to the specified UFS on this worker
```
Comment (calvinjia) [inline docs/en/Metrics-System.md]: Do we no longer keep track of the netty metrics?

Comment (aaudiber) [inline docs/en/Metrics-System.md]: Yep, we no longer track them

File: core/server/worker/src/main/java/alluxio/worker/netty/DataServerUfsFileWriteHandler.java (hunk 1 above)
Comment (aaudiber) [inline core/server/worker/src/main/java/alluxio/worker/netty/DataServerUfsFileWriteHandler.java]: done

**Comment by aaudiber** [comment]
@calvinjia Comments addressed, PTAL

**Pull request merged**